### DISCUSSION TITLE
Wrong account linked with University account for GitHub education benefits

### DISCUSSION THREAD

**Discussion by captain0jay** [community/community#51480] [GitHub Education]
I feel stupid after doing this, but i added my university account to one of my account that isn't official account of mine. Now when i wanted to add my university account to my official account GitHub shows that my university account is used already. Now i want to use my official account and get University account benefits like octernship and GitHub education pack. Can anybody suggest a solution?

**Comment by samus-aran** [comment]
Hi @captain0jay :wave:!

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.
